guides/v2.3/rest/tutorials/inventory/create-customer.md

```diff
@@ -185,7 +185,7 @@ The customer `id` is `3`.
 
 To generate a customer's access token, you must specify the customer's username and password in the payload. You do not specify an admin {% glossarytooltip 34ecb0ab-b8a3-42d9-a728-0b893e8c0417 %}authorization{% endglossarytooltip %} token.
 
-By default, a customer token is valid for 1 hour. To change this value, click **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
+By default, a customer token is valid for 1 hour. To change this value, click **Stores** > Settings > **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
```

Comment:
```suggestion
By default, a customer token is valid for 1 hour. To change this value, click **Stores** > **Settings** > **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
```

Reply:
Some maintainers are removing bolding for second point "Settings" , e.g:
https://github.com/magento/devdocs/pull/3969/commits/3c5b3b80284ef522abf3f01ca742d41114396994
That's why I try do the same.

Reply:
Good. Only bold for things you have to click on.